model/app.py

```diff
@@ -454,8 +459,15 @@ def __init__(self, model, message, cache=None):
                 self.flux = solution.fluxes
                 self.growth = self.flux[MODEL_GROWTH_RATE[model.id]]
         except OptimizationError:
+            # id model is infeasible, return the average measured fluxes only
+            logger.info('infeasible model, returning measured fluxes only')
+            changes = model.notes['changes']
             self.flux = {}
             self.growth = 0.0
+            if 'measured' in changes:
+                ids_measured_reactions = set(rxn['id'] for rxn in changes['measured']['reactions'])
+                self.flux = dict((rxn.id, (rxn.upper_bound + rxn.lower_bound) / 2)
```

Comment:
if fva-* methods are called the resulting dict wouldn't match the interface, should be `{"lower_bound": rxn.lower_bound, "upper_bound": rxn.upper_bound}` for fva